lib/portage/__init__.py

```diff
@@ -729,13 +729,17 @@ def _get_target(self):
                 cmd = [
                     BASH_BINARY,
                     "-c",
-                    (
-                        "cd %s ; git describe --match 'portage-*' || exit $? ; "
-                        + 'if [ -n "`git diff-index --name-only --diff-filter=M HEAD`" ] ; '
-                        + "then echo modified ; git rev-list --format=%%ct -n 1 HEAD ; fi ; "
-                        + "exit 0"
```

Comment:
I don't mind either way but wouldn't it be simpler to just remove the plus operators in this case? I don't know the technical details but I suppose having a single static string would still be faster than joining.

Reply:
We could do the backslash approach. Would this be a good option?
```python
Python 3.9.9 (main, Mar 17 2022, 00:21:29) 
[GCC 11.2.1 20220115] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> cmd = "cd /path/to/portage ; git describe --match 'portage-*' || exit $? ; "\
...       'if [ -n "`git diff-index --name-only --diff-filter=M HEAD`" ] ; '\
...       "then echo modified ; git rev-list --format=%%ct -n 1 HEAD ; fi ; "\
...       "exit 0"
>>> print(cmd)
cd /path/to/portage ; git describe --match 'portage-*' || exit $? ; if [ -n "`git diff-index --name-only --diff-filter=M HEAD`" ] ; then echo modified ; git rev-list --format=%%ct -n 1 HEAD ; fi ; exit 0
```

Reply:
Hmm but why not just:

```
cmd = (
    "first line"
    'second line'
    "third line"
)
```

(or whatever formatting black would use)

Reply:
Oh yeah! I forgot about that feature. I'll get on that.

Reply:
Oh, that's what it was previously. Alright, that's my bad. Thank you @mgorny!

Reply:
The adjustment was added in modernizing string patch (without the pluses, not like previously 😊).